PSKoans/Data/Advice/OOP/OOPIntroduction.Advice.json

```diff
@@ -0,0 +1,49 @@
+{
+    "Title": "Object-Orientated Programming (OOP) Introduction",
```

Comment:
How does this look when called from the `Show-Advice` cmdlet? I think this might be a bit long for this kind of thing. 

Probably the main points that need to be mentioned are:

- Everything in PowerShell is an object
- Different types of objects have different properties and methods
- `Get-Member` can be used to discover what the available properties or methods are for any given object.
- Links to OOP concepts and maybe the main .NET Core docs page: https://docs.microsoft.com/en-us/dotnet/api/?view=netcore-3.0

I think more than that starts to get to the point of being a bit too lengthy for advice snippets. 🙂

Reply:
Looks alright, if a bit long, when i run it in VS Code.

Anywho I've trimmed it down and linked to a articles explaining the basics of OOP.